source/fundamentals/monitoring.txt

```diff
@@ -65,24 +65,24 @@ You can subscribe to any of the following SDAM events:
      - Created when the topology changes, such as an election of a new
        primary or a **mongos** proxy disconnecting.
 
-       *Not applicable to a single instance.*
+       Not applicable to a `standalone instance <https://docs.mongodb.com/manual/reference/glossary/#term-standalone>`__.
```

Comment:
You can actually remove all of these, with the unified topology these events are still emitted even when connecting to `TopologyType` `Single`